students/ericstreit/lessons/lesson05/assignment/database.py

```diff
@@ -0,0 +1,217 @@
+                 # yup! you were right!
+                 # CONVERT ROW DATA TO DICT
+
+                for row in products_contents:
+                     # this could be made into a func couldn't it?
+                    logging.info(f'IMPORTING PRODUCTS DATA')
+                    convert_to_dict = {'product_id': row[0],
+                                       'description': row[1],
+                                       'product_type': row[2],
+                                       'quantity_available': row[3]}
+                    try:
+                        result = products.insert_one(convert_to_dict)
+                    # who needs comments when your logging explains it all LOL
+                        logging.info(f'Adding {convert_to_dict} to the products collection')
+                        record_count += 1
+                    except Exception as e:
+                        logging.error(f'Error adding {convert_to_dict} to the products collection!')
+                        error_count += 1
+                for row in customers_contents:
+                    logging.info(f'IMPORTING CUSTOMERS DATA')
+                    convert_to_dict = {'user_id': row[0],
+                                       'name': row[1],
+                                       'address': row[2],
+                                       'zip_code': row[3],
+                                       'phone_number': row[4],
+                                       'email': row[5]}
+                    try:
+                        result = customers.insert_one(convert_to_dict)
+                        logging.info(f'Adding {convert_to_dict} to the customers collection')
+                        record_count += 1
+                    except Exception as e:
+                        logging.error(f'Error adding {convert_to_dict} to the customers collection!')
+                        error_count += 1
+                for row in rentals_contents:
+                    logging.info(f'IMPORTING RENTALS DATA')
+                    convert_to_dict = {'product_id': row[0],
+                                       'user_id': row[1]}
+                    try:
+                        result = rentals.insert_one(convert_to_dict)
+                        logging.info(f'Adding {convert_to_dict} to the rentals collection')
+                        record_count += 1
+                    except Exception as e:
+                        logging.error(f'Error adding {convert_to_dict} to the rentals collection!')
+                        error_count += 1
+        except Exception as e:
+            logging.error(f'Hm, we had an error here?')
+            logging.error(e)
+
+    print(f' the number of records in the products collection is: {db.products.count()}')
+    print(f' the number of records in the customers collection is: {db.customers.count()}')
+    print(f' the number of records in the rentals collection is: {db.rentals.count()}')
+    return (record_count, error_count)
```

Comment:
This is a awful lot for one function. It could, and probably should, be broken into more functions. There is actually quite a bit of repeat code here that could be reduced.